pkg/virt-controller/services/renderresources.go

```diff
@@ -217,8 +217,8 @@ func WithCPUPinning(cpu *v1.CPU, annotations map[string]string) ResourceRenderer
 			emulatorThreadCPUs := resource.NewQuantity(1, resource.BinarySI)
 
 			limits := renderer.calculatedLimits[k8sv1.ResourceCPU]
-			_, EmulatorThreadCompleteToEvenParityAnnotationExists := annotations[v1.EmulatorThreadCompleteToEvenParity]
-			if EmulatorThreadCompleteToEvenParityAnnotationExists &&
+			_, emulatorThreadCompleteToEvenParityAnnotationExists := annotations[v1.EmulatorThreadCompleteToEvenParity]
+			if emulatorThreadCompleteToEvenParityAnnotationExists &&
```

Comment:
This change does not belong on this commit.

Reply:
DONE